source/pages/Home/Home.tsx

```diff
@@ -119,9 +119,7 @@ export const Home = () => {
                 </p>
               </div>
 
-              <p id="fiat-ammount">
-                {isTestnet || isLoadingBalances ? null : fiatPriceValue}
-              </p>
+              <p id="fiat-ammount">{isTestnet ? null : fiatPriceValue}</p>
```

Comment:
@lucasgabrielgsp will this need to be modified within your changes in the testnet PR?

Reply:
@lucaslovato no, it's not necessary. In testnet PR, I didn't change this structure. Is exactly the same